.github/workflows/auth-server-deploy.yml

```
@@ -0,0 +1,143 @@
+    runs-on: ubuntu-latest
+    # One promotion at a time, and every merge still gets one. Two merges landing together
+    # would otherwise race, and the loser could point prod at the OLDER version — the channel
+    # takes whichever push finishes last, not whichever commit is newer. Queued rather than
+    # cancelled: a superseded deploy is still a version somebody has to be able to find, and
+    # cancelling mid-push can leave a version uploaded but never promoted. `queue: max` is
+    # what makes that true — the default (`queue: single`) holds ONE run pending, so a third
+    # merge arriving evicts the second and that commit is never deployed at all, which is the
+    # same lost deploy the group was added to prevent. Up to 100 may wait, in arrival order.
+    concurrency:
+      group: substrat-deploy-auth-server-prod
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)